grabbit/src/main/groovy/com/twcable/grabbit/server/services/impl/DefaultServerService.groovy

```diff
@@ -71,11 +75,14 @@ class DefaultServerService implements ServerService {
                 nodeIterator = TreeTraverser.nodeIterator(rootNode)
             }
 
+            //Iterator wrapper for excludePaths exclusions
+            nodeIterator = new JcrContentExclusionIterator(nodeIterator, excludePaths)
+
             ServerBatchJob batchJob = new ServerBatchJob.ConfigurationBuilder(configurableApplicationContext)
-                .andConfiguration(new NamespaceHelper(session).namespaces.iterator(), nodeIterator, servletOutputStream)
-                .andPath(path)
-                .andContentAfterDate(afterDateString)
-                .build()
+                    .andConfiguration(new NamespaceHelper(session).namespaces.iterator(), nodeIterator, servletOutputStream)
+                    .andPath(path, excludePaths.join("*"))
```

Comment:
The `excludePaths` should be passed in as a `Collection<String>` and let the Batch Job Builder deal with "serializing" it to a `String` for better cohesion.